### DISCUSSION TITLE
chat gpt not signing in

### DISCUSSION THREAD

**Discussion by Nnr2023** [GorvGoyl/ChatGPT-Writer#79] [Report Bugs and Issues]
Since almost 15 days my chatgpt is not getting logged in. Don't know the reason why?

**Comment by tszumowski** [comment]
I second this. I am logged in to my ChatGPT Plus account. But I cannot get this app to generate. I get the error:
> There was an unknown error. Make sure you are logged-in to [chat.openai.com](https://chat.openai.com/)
